src/main/java/Tasklist.java

```diff
@@ -0,0 +1,146 @@
+import java.rmi.NotBoundException;
+import java.util.ArrayList;
+
+// Import Task class and subclasses
+import tasks.Task;
+import tasks.Todo;
+import tasks.Deadline;
+import tasks.Event;
+
+// Import custom exceptions
+// Description exceptions
+import exceptions.descriptions.EmptyDescriptionException;
+import exceptions.descriptions.IncompleteDescriptionException;
+import exceptions.descriptions.WrongFormatDescriptionException;
+
+
+/**
+ * Tasklist class
+ * Store and manage tasks.
+ */
+public class Tasklist {
+    // ATTRIBUTES
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    // METHODS
+    /**
+     * Create tasks.Todo object
+     *
+     * @param input: tasks.Todo details
+     */
+    public static Todo createTodo(String input) throws EmptyDescriptionException {
+        // Check if input is empty
+        if (input.isEmpty()) {
+            throw new EmptyDescriptionException();
+        }
+
+        return new Todo(input);
+    }
+
+    /**
+     * Create tasks.Deadline object
+     *
+     * @param input: tasks.Deadline details
+     */
+    public static Deadline createDeadline(String input) throws EmptyDescriptionException, WrongFormatDescriptionException, IncompleteDescriptionException {
+        // Check if input is empty or wrong format
+        if (input.isEmpty()) {
+            throw new EmptyDescriptionException();
+        } else if (!input.contains("/by")) {
+            throw new WrongFormatDescriptionException("Deadline");
+        }
+
+        int byIndex = input.indexOf("/by");
+        int byLength = "/by".length();
+        String description = input.substring(0, byIndex).strip();
+        String by = input.substring(byIndex + byLength).strip();
+
+        // Check if description/by is empty
```

Comment:
this comment is a bit redundant, it can already be inferred by the code 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#do-not-repeat-the-obvious